vellum/workflows/triage_message/nodes/triage_message_node.py

```diff
@@ -113,6 +114,29 @@ def job_opportunity_response(
     pass
 
 
+def create_meeting(
+    scheduling_link: str,
+):
+    """
+    Schedule a meeting using a scheduling link (Cal.com, Calendly, etc.).
+    
+    Use this function when:
+    - The message contains a scheduling link (e.g., cal.com, calendly.com, etc.)
+    - Someone is asking you to schedule a meeting or book time
+    - A link is provided for booking an appointment
+    
+    To detect scheduling links, look for:
+    - URLs containing "cal.com", "calendly.com", "reclaim.ai", "motion.com"
+    - Phrases like "book a time", "schedule a meeting", "pick a time", "here's my calendar"
+    - Links in the message body that appear to be for scheduling
+    
+    The system will automatically schedule the earliest available slot using the contact's
+    information from the message. After scheduling, you will be prompted again to craft
+    an appropriate response with the meeting details.
```

Comment:
Add some copy here about how if we fail to schedule, don't try again and just let the recipient know that you don't know how to schedule meetings yet and if it would be possible to continue the interview over email